### DISCUSSION TITLE
Issue with "grid-cols-10" not working in Tailwind CSS

### DISCUSSION THREAD

**Discussion by Sathishkumar0404** [tailwindlabs/tailwindcss#12878] [Help] [answered]
Hi everyone,

I'm facing an unusual issue with Tailwind CSS where the `grid-cols-10` class is not working as expected. Strangely, `grid-cols-11` works fine. I've double-checked for typos, conflicting styles, and even tried restarting my development server, but the problem persists.

Here's a simplified version of my code:

```jsx
<div className="grid gap-5 px-4 grid-cols-10">
    <label htmlFor="rules_field" className="col-span-3 text-xs mb-2 text-gry-700">
        Field
    </label>
    <label htmlFor="rules_operator" className="col-span-3 text-xs mb-2 text-gry-700">
        Operator
    </label>
    <label htmlFor="rules_value" className="col-span-3 text-xs mb-2 text-gry-700">
        Value
    </label>
</div>

**Comment by AM-NiceDay** [comment]
Hey @Sathishkumar0404!

I tried your code example in Tailwind Play, and it worked as expected - you can find it here: [https://play.tailwindcss.com/iUepkYbt2Z](https://play.tailwindcss.com/iUepkYbt2Z).

To fix the issue, please check your Tailwind Configuration and ensure the following:
-  The file containing your shared code is included in the `content` configuration space.
-  You haven't modified the `gridTemplateColumns` property in the theme.

**Reply by Sathishkumar0404** [reply]
Hey @AM-NiceDay ,

Thanks for your reply!

I've reviewed my Tailwind Configuration based on your recommendations, and everything seems to be in order. However, I'm still encountering issues with `grid-cols-10`, `grid-cols-9`, `grid-cols-8`, and `grid-cols-7` classes not working, while others are functioning correctly.

Here Configurations:

- Next.js version: 13.1.0
- Tailwind CSS version: 3.3.2
- `tailwind.config.js`:
```javascript
module.exports = {
  content: [
    './pages/**/*.{js,ts,jsx,tsx,mdx}',
    './components/**/*.{js,ts,jsx,tsx,mdx}',
    './app/**/*.{js,ts,jsx,tsx,mdx}'
  ],
  theme: {
    extend: {
      backgroundImage: {
        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
      },
      colors: {
        primary: '#635DFF',
        text_primary: '#1E212A',
        text_secondary: '#423fa1',
        secondary: '#564ed1',
        bg_primary: '#e9e8ff',
        bg_secondary: '#EFF0F2',
        ringprimary: '#3b82f680',
        primary_black: '#1E212a',
        bg_primary: '#f9f9f9',
      }
    }
  },
  plugins: []
};
```
- `postcss.config.js`:
```javascript
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
}
```

I'm still unsure why these specific classes aren't working as expected. Any further insights or troubleshooting steps you can provide would be greatly appreciated!

Thanks again for your assistance!

**Reply by inlancersystem4** [reply]
https://play.tailwindcss.com/X43M8bndhy

**Reply by AM-NiceDay** [reply]
Can you please provide the relative path to the source file where you are using `grid-cols-10`?

**Reply by Sathishkumar0404** [reply]
Sure, The file located at ''./components/FormArrayComponent.jsx''.

**Reply by Sathishkumar0404** [reply, marked as answer]
The issue has been resolved by adding the file path to the `tailwind.config.js` file.

The main component where I was encountering the error is located in the `containers` directory. By adding the following path to the Tailwind CSS configuration, the problem was fixed:

```javascript
'./containers/**/*.{js,ts,jsx,tsx,mdx}',

**Reply by sir-stevie** [reply]
Hello, I am having the same issue with the `grid` class, `tailwindcss@4` which doesn't use a config file. Anybody got an idea how I can work around this?

**Comment by inlancersystem4** [comment]
> Hey @Sathishkumar0404!
> 
> I tried your code example in Tailwind Play, and it worked as expected - you can find it here: https://play.tailwindcss.com/iUepkYbt2Z.
> 
> To fix the issue, please check your Tailwind Configuration and ensure the following:
> 
> * The file containing your shared code is included in the `content` configuration space.
> * You haven't modified the `gridTemplateColumns` property in the theme.
